src/app/routes/article/getInitialData/index.ts

```diff
@@ -14,13 +13,6 @@ import getOnwardsPageData from '../utils/getOnwardsData';
 
 const logger = nodeLogger(__filename);
 
-const removeAmp = (path: string) => path.split('.')[0];
-const getOptimoId = (path: string) => path.match(/(c[a-zA-Z0-9]{10}o)/)?.[1];
-const getCpsId = (path: string) => path;
-
-const getId = (pageType: string) =>
-  pipe(getUrlPath, removeAmp, pageType === 'article' ? getOptimoId : getCpsId);
```

Comment:
This logic is now in the BFF module https://github.com/bbc/fabl-modules/pull/3740/commits/d08adb395f50b83f3cc828411c6803997d369e87